packages/ui/src/Customization/CustomizationsExtensionDetail.tsx

```diff
@@ -1,14 +1,249 @@
+import { Extension, Integration } from '@syndesis/models';
+import { OptionalIntUtils } from '@syndesis/utils';
+import {
+  Button,
+  Card,
+  CardBody,
+  CardHeading,
+  CardTitle,
+  OverlayTrigger,
+  Row,
+  Table,
+  Tooltip,
+} from 'patternfly-react';
 import * as React from 'react';
+import './CustomizationsExtensionDetail.css';
 
 export interface IExtensionDetailProps {
-  extensionId: string;
-  i18nTitle: string;
+  extension: Extension;
+  i18nDelete: string;
+  i18nDeleteTip?: string;
+  i18nDescription: string;
+  i18nIdMessage: string;
+  i18nLastUpdate: string;
+  i18nLastUpdateDate: string;
+  i18nName: string;
+  i18nOverview: string;
+  i18nSupportedConnectors: string;
+  i18nSupportedSteps: string;
+  i18nType: string;
+  i18nTypeMessage: string;
+  i18nUpdate: string;
+  i18nUpdateTip?: string;
+  i18nUsage: string;
+  i18nUsageMessage: string;
+  onDelete: (extensionName: string) => void;
+  onSelectIntegration: (integrationId: string) => void;
+  onUpdate: (extensionName: string) => void;
+  propDescription: string;
```

Comment:
What's this field for?  I feel like this could use a better name or comment.

Reply:
Agree. I'll come up with better names for this and `propName` (as soon as I figure out what the heck those properties are:)